src/Elastic.Apm/Config/AbstractConfigurationReader.cs

```diff
@@ -651,22 +651,13 @@ protected string ParseCaptureBody(ConfigurationKeyValue kv)
 			return captureBodyInConfig.ToLowerInvariant();
 		}
 
-		protected List<string> ParseCaptureBodyContentTypes(ConfigurationKeyValue kv, string captureBody)
+		protected List<string> ParseCaptureBodyContentTypes(ConfigurationKeyValue kv)
```

Comment:
Sooo..what happens here is that since `CAPTURE_BODY` is now live changeable, we can't take that into consideration when we read `CaptureBodyContentTypes`, right?

Reply:
That is correct.